servers/visual/visual_server_viewport.cpp

```diff
@@ -178,11 +178,14 @@ void VisualServerViewport::_draw_viewport(Viewport *p_viewport, ARVRInterface::E
 		VSG::rasterizer->restore_render_target();
 
 		if (scenario_draw_canvas_bg && canvas_map.front() && canvas_map.front()->key().layer > scenario_canvas_max_layer) {
+			Ref<ARVRInterface> arvr_interface = ARVRServer::get_singleton()->get_primary_interface();
```

Comment:
Note that the changes I've done here aren't really related anymore to this PR, maybe I should split them out, but it turns out I never implemented the ARVR stuff in each option where we render our 3D scene.